src/main/java/com/google/opensesame/servlets/ProjectServlet.java

```diff
@@ -42,7 +43,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       return;
```

Comment:
I see that you used the ```getParameter``` function in your ```doGet```. does it do anything in this case?

Reply:
Good point haha. It doesn't really in this case, does it? I'd like to keep it there for the time being in the case that I add more parameters down the line that might need different behaviors, and in that case I'd like things to stay consistent with this `getParameter` as well. Good catch